Makefile

```diff
@@ -43,3 +43,8 @@ fmt:
 
 .PHONY: check
 check: errcheck vet fmt test-unit
+
+.PHONY: integration
+integration:
+	go build -o mobile ./cmd/mobile
```

Comment:
I think we probably should take advantage of the Makefile's ability to run setup steps, i.e
```
integration: build
    go test ./integration
```

this change is not required for the pr.